elasticsearch-akka/src/main/scala/com/sumologic/elasticsearch/akkahelpers/BulkIndexerActor.scala

```diff
@@ -63,8 +63,8 @@ class BulkIndexerActor(restlasticSearchClient: RestlasticSearchClient, bulkConfi
       queue = OpWithTarget(BulkOperation(index, Some(indexName -> tpe), doc), sender(), sess) :: queue
       if (queueFull) flush()
 
-    case UpdateRequest(sess, index, tpe, doc) =>
-      queue = OpWithTarget(BulkOperation(update, Some(index -> tpe), doc), sender(), sess) :: queue
+    case UpdateRequest(sess, index, tpe, doc, retryOnVersionConflictOpt) =>
```

Comment:
nit: may be rename this to `retryOnConflictOpt` to be more like the ES parameter?